blueprints/ember-redux-shim/index.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
       .then(() => {
         return this.addAddonsToProject({
           packages: [
-            {name: 'ember-lodash-shim', target: '^1.2.0'},
+            {name: 'ember-lodash-shim', target: '^2.0.0'},
```

Comment:
This repo works with both versions so I'm not too worried, a consumer can continue to use `ember-lodash-shim` 1.x if they so choose.